pkgs/development/compilers/dotnet/dotnet-sdk-setup-hook.sh

```diff
@@ -60,7 +64,7 @@ configureNuget() {
         done
     fi
 
-    if [[ -f paket.dependencies ]]; then
+    if [[ -z ${keepNugetConfig-} && -f paket.dependencies ]]; then
```

Comment:
Also needs to be documented (I know it already existed, but I forgot to mention this when it was added)

Reply:
It doesn't really fit in the `buildDotnetModule` docs, because you shouldn't have to worry about it there, but i do think there could be a whole section about the nuget package and dotnet-sdk hooks, and the variables they use: `linkNugetPackages`, etc.